Source: maxice8
Upstream: not upstreamable
Reason: Fixes building of package.

ERROR: 
Error in gtkdoc helper script:
ERROR: 'gtkdoc-mkhtml' failed with status 1
Traceback (most recent call last):
  File "/usr/bin/gtkdoc-mkhtml", line 51, in <module>
    sys.exit(mkhtml.run(options))
  File "/usr/share/gtk-doc/python/gtkdoc/mkhtml.py", line 93, in run
    quiet] + remaining_args + [gtkdocdir + '/gtk-doc.xsl', document])
  File "/usr/share/gtk-doc/python/gtkdoc/mkhtml.py", line 42, in run_xsltproc
    return subprocess.call(command + args)
  File "/usr/lib/python2.7/subprocess.py", line 172, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib/python2.7/subprocess.py", line 394, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1047, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory

--- meson.build
+++ meson.build
@@ -144,6 +144,6 @@ if libdrm_intel.found()
 endif
 subdir('man')
 # has_exe_wrapper() is undefined if building natively
-if not meson.is_cross_build() or not meson.has_exe_wrapper()
-	subdir('docs')
-endif
+# if not meson.is_cross_build() or not meson.has_exe_wrapper()
+#  	subdir('docs')
+# endif
